packages/server/api/src/app/flows/flow/flow.controller.ts

```diff
@@ -197,6 +252,33 @@ async function extractUserIdFromPrincipal(
   return project.ownerId;
 }
 
+async function validateTriggerType(
```

Comment:
If anyone knows an easier way to verify that the trigger is of strategy polling, lmk